sonic-xcvrd/scripts/xcvrd

```diff
@@ -380,28 +384,36 @@ def post_port_dom_info_to_db(logical_port_name, table, stop_event=threading.Even
 # Update port dom/sfp info in db
 def post_port_sfp_dom_info_to_db(is_warm_start, stop_event=threading.Event()):
     # Connect to STATE_DB and create transceiver dom/sfp info tables
-    transceiver_dict = {}
-    state_db = daemon_base.db_connect("STATE_DB")
-    int_tbl = swsscommon.Table(state_db, TRANSCEIVER_INFO_TABLE)
-    dom_tbl = swsscommon.Table(state_db, TRANSCEIVER_DOM_SENSOR_TABLE)
-
-    appl_db = daemon_base.db_connect("APPL_DB")
-    app_port_tbl = swsscommon.ProducerStateTable(appl_db,
-                                                 swsscommon.APP_PORT_TABLE_NAME)
+    transceiver_dict, state_db, appl_db, int_tbl, dom_tbl, app_port_tbl = {}, {}, {}, {}, {}, {}
+
+    # Get the namespaces in the platform
+    namespaces = multi_asic.get_namespaces()
+    for namespace in namespaces:
+        asic_id = multi_asic.get_asic_id_from_namespace(namespace)
```

Comment:
could you point me the design for multi basic?  From code here my understanding is that each ASIC "own" a set of ports/transceiver, I would like to understand more.

Reply:
> could you point me the design for multi basic? From code here my understanding is that each ASIC "own" a set of ports/transceiver, I would like to understand more.

The design is explained in brief in the multi-ASIC HLD, but I have an internal document on the design approaches, shall share it with you.

Reply:
I have moved the design into a separate design doc, https://github.com/judyjoseph/SONiC/blob/multi_asic/doc/pmon/pmon_multiasic_design.md. Let me know if you are not able to access.

Reply:
@keboliu, were you able to go through the approach doc ? (https://github.com/Azure/SONiC/pull/655) 
Could you share your comments.